### PR TITLE
gitman v1.0 (new formula) - Initial commit

### PR DESCRIPTION
**Pull request by jonstaff**
Initial commit for Gitman v1.0, a simple tool to help manage Git repos.

### CODE REVIEW

File: Library/Formula/gitman.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Gitman < Formula
+  homepage "https://github.com/jonstaff/gitman"
+  url "https://github.com/jonstaff/gitman/archive/v1.0.tar.gz"
+  version "1.0"
```
Comment (DomT4) [inline Library/Formula/gitman.rb]: Can remove this line.

Comment (jonstaff) [inline Library/Formula/gitman.rb]: I'm not sure I follow; should this line be removed? It was generated.

Comment (DomT4) [inline Library/Formula/gitman.rb]: Aye. The ` version ` line is safe to remove in this case - ` brew ` can automatically read a version from a relatively simple URL like this.

**Comment by jonstaff** [comment]
@DomT4 The `brew audit` error makes sense now.  I updated the PR with that fix.

**Comment by DomT4** [comment]
Thanks! Should pass the test-bot now.

I don't know if this will qualify to be accepted into Homebrew at this point, our [Acceptable Formula](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Acceptable-Formulae.md#niche-or-self-submitted-stuff) guide tends to frown on self-submissions of new projects unless they're very popular. One of the maintainers for Homebrew/homebrew will make that decision one way or another though - I'm only involved in a few of the core taps.

**Comment by mikemcquaid** [comment]
This project is a 3 hour old self-submission so passing on it.

**Pull request closed without merging**